SPECS/microsemi-aacraid.spec

```diff
@@ -19,10 +19,12 @@
 Summary: %{vendor_name} %{driver_name} device drivers
 Name: %{vendor_label}-%{driver_name}
 Version: 1.2.1.60001
-Release: %{?xsrel}%{?dist}
+Release: %{?xsrel}.0.fixSeries6.1%{?dist}
 License: GPL
 Source0: microsemi-aacraid-1.2.1.60001.tar.gz
 
+Patch0: aacraid-1.2.1.60001-fix-Series-6.patch
```

Comment:
Adding the mention  that it's a patch from the XCP-ng project, not from upstream packaging, and thus numbering it Patch1000. I also added the `.XCP-ng` suffix to the file name to differentiate it from a backport.

```suggestion
# XCP-ng Patches
Patch1000: aacraid-1.2.1.60001-fix-Series-6.XCP-ng.patch
```